broker/controller.go

```diff
@@ -197,6 +201,44 @@ func (c *Controller) CreateServiceInstance(w http.ResponseWriter, r *http.Reques
 	WriteResponse(w, http.StatusAccepted, response)
 }
 
+func (c *Controller) UpdateServiceInstance(w http.ResponseWriter, r *http.Request) {
+	log.Println("Update Service Instance...")
+
+	var instance ServiceInstance
+
+	err := ProvisionDataFromRequest(r, &instance)
+	if err != nil {
+		log.Println("Provision instatnce:", err)
+	}
+
+	instanceID := ExtractVarsFromRequest(r, "service_instance_guid")
+
+	skipS3 := true
+
+	err = c.DeployCluster(instance, &skipS3, instanceID)
```

Comment:
are you sure `DeployCluster `?
now both `UpdateServiceInstance` and `CreateServiceInstance` looks identical. is it ok?